Dockerfile.test

```diff
@@ -9,13 +9,13 @@ MAINTAINER Richard Knop <risoknop@gmail.com>
 ENV PATH /go/bin:$PATH
 
 # Cd into the source code directory
-WORKDIR /go/src/github.com/RichardKnop/machinery
+WORKDIR /go/src/github.com/Michael-LiK/machinery
```

Comment:
probably shouldn't merge your name into the main repo @Michael-LiK

Reply:
if you're using this for local dev, check out `replace` syntax in mode files

Reply:
OK THANKS